src/core/particle.cpp

```diff
@@ -909,12 +909,16 @@ float Particle::ReturnLogLikelihood(Image &input_image, CTF &input_ctf, Reconstr
 //	particle_image->QuickAndDirtyWriteSlice("diff.mrc", 1);
 	// This low-pass filter reduces the number of independent pixels. It should therefore be applied only to
 	// the reference (temp_projection), and not to the difference (temp_particle - temp_projection), as is done here...
-	if (classification_resolution_limit > 0.0)
+//	temp_particle->ForwardFFT();
+//	if (classification_resolution_limit < 20.0f) temp_particle->CosineMask(original_pixel_size / 20.0f, original_pixel_size / 10.0f, true);
```

Comment:
Does this just add two commented out lines?

Reply:
Yes, Niko was working on something (pre-pandemic) but wasn't quite sure where he left off. He wanted to leave the comments as a reminder.

Reply:
@timothygrant80 Here is a larger view of that block from this commit (first) and from 1.0 (second) - it does look like just some extra comments. 

https://github.com/timothygrant80/cisTEM/blob/eb037f1b23cc359e6dfd2f8d7012e347ae74da5e/src/core/particle.cpp#L908-L921

https://github.com/timothygrant80/cisTEM/blob/c11ab49a442153734e3a7acaabc0b0fd80809c54/src/core/particle.cpp#L690-L699